heimlig/src/client/api.rs

```diff
@@ -52,6 +51,7 @@ impl<
         self.send_request(request).await
     }
 
+    /// Generate an symmetric key pair and store it in the HSM.
```

Comment:
Does it make sense to add `missing_docs` in general to the project :thinking: ?

Reply:
I think we should once we move towards version 1.0. Right now, the code is changing a little too fast.